code/modules/reagents/chemistry/reagents/misc.dm

```diff
@@ -39,6 +39,10 @@
 	reagent_state = GAS
 	color = "#808080" // rgb: 128, 128, 128
 
+/datum/reagent/oxygen/on_mob_life(var/mob/living/carbon/human/H)
```

Comment:
Again, this is snowflaked---this should be handled on the species reagents procs.